### PR TITLE
Ability to disable a shortcut

### PR DESCRIPTION
**Pull request by j0k3r**

### CODE REVIEW

File: spec/keyboard-commands.spec.js [hunk 1]
```diff
@@ -110,5 +110,30 @@ describe('KeyboardCommands TestCase', function () {
             });
             expect(editor.execAction).not.toHaveBeenCalled();
         });
+
+        it('should not execute when the command key are false', function () {
```
Comment (nmielnik) [inline spec/keyboard-commands.spec.js]: Can you add another test case (or another assertion in this test case) where if `event.shiftKey` is `true`, it should NOT block the code from executing?

Comment (j0k3r) [inline spec/keyboard-commands.spec.js]: Like that ?

```js
fireEvent(editor.elements[0], 'keydown', {
    keyCode: 'J'.charCodeAt(0),
    ctrlKey: true,
    metaKey: true,
    shiftKey: true
});
expect(editor.execAction).not.toHaveBeenCalled();
```

Comment (nmielnik) [inline spec/keyboard-commands.spec.js]: Actually, you could look at the result of `fireEvent()` and see if `preventDefault()` had been called on it.

That way you could:
```js
var result = fireEvent(... {
    ...
    shiftKey: false
});
expect(result).toBe(true); // true == cancelled

result = fireEvent(... {
    ...
    shiftKey: true
});
expect(result).toBe(false); // false == not cancelled
```

I think that's a better way to check if the functionality does what you expect it to do, what do you think?

Comment (j0k3r) [inline spec/keyboard-commands.spec.js]: It's a good idea :bulb:

**Pull request merged**